docs/themes.md

```diff
@@ -43,3 +43,12 @@ add_theme_support( 'editor-color-palette',
 ```
 
 The colors will be shown in order on the palette, and there's no limit to how many can be specified.
+
+### Disabling custom colors in block Color Palettes
+
+By default, the color pallet offered to blocks, allows the user to select a custom color different from the editor or theme default colors.
+Themes can disable this feature using:
+```php
+add_theme_support( 'disable-custom-colors', true );
+```
+This flag will make sure users are only able to choose colors from the editor-color-pallet the theme provided or from the editor default colors if the theme did not provide one.
```

Comment:
Can we wrap editor-color-pallet with '`' to better stand out?

Reply:
In order to create this stand out effect I had to remove ```php wrapping but as it is just for a flag I think it worths it.

Reply:
You didn't understand me correctly :)

I wanted only this:

> ```php
> add_theme_support( 'disable-custom-colors', true );
> ```
> This flag will make sure users are only able to choose colors from the `editor-color-palette` the theme provided or from the editor default colors if the theme did not provide one.

Reply:
Ah, sorry the misunderstanding, it was updated.